small-talk/util/smolCwyptoTEST.mjs

```diff
@@ -0,0 +1,25 @@
+import { createCookieHash } from "@/util/smolCwypto.js";
+
+describe('createCookieHash', () => {
+    it('should create a valid hash', () => {
+        const data = { username: 'user123', timestamp: Date.now() };
+        const secretKey = 'mySecretKey';
+
+        const hash = createCookieHash(data, secretKey);
+
+        // Add your assertions here
+        expect(hash).to.be.a('string');
+        expect(hash).to.have.lengthOf(64); // SHA-256 hash length
```

Comment:
Maybe a redundant test, but would it be worth it to verify that we can repeat the hash functionality? Make sure that if someone puts in the same password, it should return the same hash function twice?  Maybe a useless test case, but it came to mind.

Reply:
Maybe useless but just a test case I thought of.

Reply:
Or even that a change in the secret key should change the hash